### ISSUE TITLE
Open Source version of Sound Seeder App

### ISSUE THREAD

**Issue by karuppiah7890** [mikaelbr/open-source-ideas#28]
## Project description

I want to make an Open Source version of this App : 
https://play.google.com/store/apps/details?id=com.kattwinkel.android.soundseeder.player&hl=en

For reasons similar to this issue and this issue is a related issue :
https://github.com/mikaelbr/open-source-ideas/issues/27

Both issues are related problems if you check out the Apps. It's just that in SoundSeeder App, the Android device is the source of the audio and all other devices including Android can act as speakers. Unlike in SoundWire App, where only the PC/Raspberry Pi is the source of the audio.

## Relevant Technology

Platform : Android
Language : Java
Concepts to know : Networking. Streaming, especially Audio Streaming.

## Who is this for

Experienced developers.

---

Share on Twitter/other social media if you will. Something like

```
New project on Open Source Ideas Initiative: 
Open Source version of Sound Seeder App - 
```

**Comment by prahladyeri** [comment]
Nice idea. Has anyone started out on this? Found out what android SDK APIs you need to create this system?

**Comment by mikaelbr** [comment]
As far as I've heard, no one has started working on this. 👍

**Comment by prahladyeri** [comment]
@mikaelbr In that case, I'd like to take this as a side project. Should I just start a new github repo and link it here, or are there any other formalities to be done?

**Comment by mikaelbr** [comment]
That would work just great.

**Comment by karuppiah7890** [comment]
@prahladyeri I could discuss about this idea if you want, but I don't know much about the new Android APIs like the ones in L, M, N, O. I used to code in Android and I have experience with only some of the SDKs till KitKat and I have done some socket programming in Java in Android Apps.

**Comment by buddhashrestha** [comment]
@prahladyeri have you had any progress with this?

**Comment by prahladyeri** [comment]
@buddhashrestha Yep, I'm working on it, please give me some time. I'll upload the code and share the github link soon once I got something working.

**Comment by buddhashrestha** [comment]
@prahladyeri awesome. can't wait

**Comment by mshingote** [comment]
> @mikaelbr In that case, I'd like to take this as a side project. Should I just start a new github repo and link it here, or is there any other process to it?

Have you started on this?

**Comment by prahladyeri** [comment]
Sorry dude, I couldn't progress this further due to some other work. You are free to take this if you want.

**Comment by mshingote** [comment]
Thanks for prompt reply @prahladyeri

**Comment by mani1911** [comment]
I has the same idea since i dint have a speaker. Can someone tell me the progress of this app and the tech used?
